examples/src/app/error.tsx

```diff
@@ -0,0 +1,41 @@
+'use client'
+
+function Samp(props: { children: React.ReactNode }) {
+  return <samp className="bg-red-100 p-1 rounded-md font-bold">{props.children}</samp>
+}
+
+export default function Error(props: { error: any }) {
+  let message = <p>Check the server console for more information.</p>
+
+  const errMsg = props.error.message
```

Comment:
if we are only using this for `startsWith` calls that extract the code, why not just access the code field directly here?

Reply:
The `code` field is not available on the serialized version of the Error that Next sends down to the client, unfortunately.